src/containers/Review/ReviewPageWrapper.tsx

```diff
@@ -9,7 +13,19 @@ const ReviewPageWrapper: React.FC = () => {
     params: { serialNumber, reviewId },
   } = useRouter()
 
-  return (
+  // TODO: Need to wait for trigger to run that will set the Review status as DRAFT (after creation)
+
+  const { error, loading, data } = useLoadReview({ reviewId: Number(reviewId) })
+
+  useEffect(() => {
+    console.log('data', data)
+  }, [data])
+
+  return error ? (
+    <Message error header="Problem to load review" list={[error]} />
```

Comment:
Do we want to put these in messages/constants?

Reply:
Ah, yes. Good point. I'll do this for all pages right away.

Reply:
I have added as many as I could easily find  :)